### PR TITLE
Orca: Add orca ray ctx example to run on self hosted

### PR DESCRIPTION
**Pull request by aixideng**
## Description

Add orca ray ctx example to run on self hosted.

### CODE REVIEW

File: .github/workflows/nightly_test.yml [hunk 1]
```diff
@@ -152,18 +157,16 @@ jobs:
...
-
-
-
-
-
-
-
 
+  Orca-Ray-Ctx-Example:
+    # if: ${{ github.event.schedule || github.event.inputs.artifact == 'Orca-Ray-Ctx-Example' || github.event.inputs.artifact == 'all' }}
```
Comment (liu-shaojun) [inline .github/workflows/nightly_test.yml:162]: please uncomment this line

File: .github/workflows/nightly_test.yml [hunk 2]
```diff
@@ -1,7 +1,11 @@
 name: Nightly Test
 
 on:
-  #pull_request:
+  pull_request:
+    branches: [ "main" ]
+    paths:
+      - 'python/orca/**'
+      - '.github/workflows/nightly_test.yml'
```
Comment (liu-shaojun) [inline .github/workflows/nightly_test.yml:8]: please remove this pull_request action, if you want to trigger this test for PR validation, please update PR_Validation.yml to add Orca-Ray-Ctx-Example.

File: .github/workflows/PR_validation.yml [hunk 3]
```diff
@@ -26,6 +30,7 @@ jobs:
     outputs:
       friesian: ${{ steps.filter.outputs.friesian }}
       dllib: ${{ steps.filter.outputs.dllib }}
+      orca: ${{ steps.filter.outputs.dllib }}
```
Comment (liu-shaojun) [inline .github/workflows/PR_validation.yml:33]: `orca: ${{ steps.filter.outputs.dllib }}`  => `orca: ${{ steps.filter.outputs.orca }}`??

File: .github/workflows/PR_validation.yml [hunk 4]
```diff
@@ -67,4 +75,19 @@ jobs:
...
         uses: ./.github/actions/maven-setup-action
       - name: Run test
-        uses: ./.github/actions/dllib-scala-ut-action
\ No newline at end of file
+        uses: ./.github/actions/dllib-scala-ut-action
+
+
+  Orca-Ray-Ctx-Example:
+    needs: changes
+    if: ${{ needs.changes.outputs.dllib == 'true' }}
```
Comment (liu-shaojun) [inline .github/workflows/PR_validation.yml:83]: if: ${{ needs.changes.outputs.orca == 'true' }}

**Review by liu-shaojun — APPROVED** [review]
LGTM

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**